lib/orderbook/OrderBook.ts

```diff
@@ -555,6 +571,15 @@ class OrderBook extends EventEmitter {
    * @returns `false` if it's a duplicated order or with an invalid pair id, otherwise true
    */
   private addPeerOrder = (order: IncomingOrder): boolean => {
+    if (!this.thresholds.disable) {
+      const shouldApplyThreshold = this.thresholds.disableFor.includes(order.peerPubKey);
```

Comment:
The `disableFor` functionality seems unnecessary. It seems very clunky to have to put pub keys of peers in the config file. I would drop this logic altogether.